projects/driver_controls_pedal/src/direction_indicator.c

```diff
@@ -0,0 +1,38 @@
+#include "direction_indicator.h"
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_unpack.h"
+#include "exported_enums.h"
+#include "pc_input_event.h"
+
+static StatusCode prv_handle_direction(const CanMessage *msg, void *context,
```

Comment:
I thought the pedal board was supposed to be a slave only assuming your master is the center console?